pkg/api/serialization_test.go

```diff
@@ -94,6 +95,7 @@ func roundTripSame(t *testing.T, item runtime.Object, except ...string) {
 	codec, err := testapi.GetCodecForObject(item)
 	if err != nil {
 		t.Errorf("unexpected error: %v", err)
+		return
```

Comment:
Anything used in loops I like to error and let continue to see as many possible errors at once. Don't feel strongly though